lib/floe/workflow/states/task.rb

```diff
@@ -113,7 +114,7 @@ def process_input(input)
         end
 
         def process_output!(results)
-          output = process_input(context.state["Input"])
+          output = context.input.dup
```

Comment:
We still need to use `process_input` or something like it to apply the `InputPath` and the `Parameters`

Reply:
I updated the description above (to include some of the commit comments)

From what I can tell, the output is based upon the raw input and not the processed input.